shop.html

```diff
@@ -0,0 +1,599 @@
+              accessories to enhance your journey and streamline your adventures
+            </p>
+          </a>
+          <a href="#" class="group">
+            <div
+              class="aspect-h-1 aspect-w-1 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2"
+            >
+              <img
+                src="/assets/images/kuiye-chen-NuOGFo4PudE-unsplash 1.png"
+                alt="Paper card sitting upright in walnut card holder on desk."
+                class="h-full w-full object-cover object-center group-hover:opacity-75"
+              />
+            </div>
+            <div class="mt-4 flex items-center justify-between">
+              <h3 class="text-travel-orange text-xl font-semibold">
+                Accessories
+              </h3>
+            </div>
+            <p class="mt-1 text-sm font-medium">
+              Explore our accessory section for the perfect finishing touch to
+              elevate your style
+            </p>
+          </a>
+          <a href="#" class="group">
+            <div
+              class="aspect-h-1 aspect-w-1 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2"
+            >
+              <img
+                src="/assets/images/curology-wK0h-mlvfuc-unsplash.png"
+                alt="Textured gray felt pouch for paper cards with snap button flap and elastic pen holder loop."
+                class="h-full w-full object-cover object-center group-hover:opacity-75"
+              />
+            </div>
+            <div class="mt-4 flex items-center justify-between">
+              <h3 class="text-travel-orange text-xl font-semibold">
+                Health &amp; Beauty
+              </h3>
+            </div>
+            <p class="mt-1 text-sm font-medium">
+              Discover our health and beauty collection for products designed to
+              enhance your well-being and radiate your natural beauty
+            </p>
+          </a>
+        </div>
+      </section>
+      <section class="relative">
+        <img
+          src="/assets/images/ross-parmly-rf6ywHVkrlY-unsplash 1.svg"
+          alt="Plane in the sky"
+          class="w-full h-auto object-cover"
+        />
+        <div
+          class="absolute inset-0 flex flex-col justify-center items-center bg-opacity-50 space-y-8"
+        >
+          <div class="max-w-5xl space-y-5 text-center">
+            <h2 class="text-5xl font-semibold">
+              Get 25% off during our one-time sale
+            </h2>
+            <p class="max-w-xl mx-auto font-semibold text-xl">
+              Most of our products are limited releases that won't come back.
+              Get your favorite items while they're in stock
+            </p>
+          </div>
+          <div class="mt-4">
+            <button
+              class="px-4 py-2 bg-travel-orange text-white font-semibold rounded-lg hover:bg-orange-600"
+            >
+              Get access to our one-time sale
+            </button>
```

Comment:
No link on button - maybe link to contact page?

Reply:
this is just a dummy button as we dont have any sales as its just a static site